external/kafka-0-10-sql/src/test/resources/kafka-source-initial-offset-version-2.1.0.bin

```diff
@@ -0,0 +1 @@
+2{"kafka-initial-offset-2-1-0":{"2":0,"1":0,"0":0}}
```

Comment:
Spark 2.1.0 doesn't output a new line, so I prefer to use the same format. I will fix it when merging your PR.